packages/docs/src/layouts/default.vue

```diff
@@ -230,6 +230,13 @@ html {
 
     overflow-y: auto;
     overflow-x: hidden;
+
+    @media(max-width:670px){
+      & > :last-child {
```

Comment:
```suggestion
      & > .layout.gutter--xl {
```

Because of this, we will not have side effects, if we want to add something to the end. And it is just clearer for humans :).